dictionaries/dict.json

```diff
@@ -1101,6 +1101,7 @@
 "A*PB/A*PB/AR/PWOR": "Ann Arbor",
 "A*PB/A*PB/ARB/O*R": "Ann Arbor",
 "A*PB/A*PB/HAEURB": "Anne Heche",
+"A*PB/HREUBGS": "Analytics",
```

Comment:
I think `A*PB` is a suffix, not a capitalised prefix: `"A*PB": "{^an}",` — am I missing something?

Reply:
Oh, I worked it out — new entry in Plover